test/functional/ui/cmdline_spec.lua

```diff
@@ -321,58 +235,39 @@ local function test_cmdline(linegrid)
       cmdline_block = {
         { { 'function Bar()' } },
       },
-    }
+    })
 
     feed('endfunction<cr>')
-    screen:expect {
-      grid = [[
-      ^                         |
-      {1:~                        }|*3
-                               |
-    ]],
-      cmdline = { { abort = false } },
-    }
+    screen:expect({ grid = empty_grid, cmdline = { { abort = false } } })
   end)
 
   it('works with cmdline window', function()
     feed(':make')
-    screen:expect {
-      grid = [[
-      ^                         |
-      {1:~                        }|*3
-                               |
-    ]],
+    screen:expect({
+      grid = empty_grid,
       cmdline = {
         {
           firstc = ':',
           content = { { 'make' } },
           pos = 4,
         },
       },
-    }
+    })
 
     feed('<c-f>')
-    screen:expect {
-      grid = [[
+    local s1 = [[
```

Comment:
Not worth it for `s1` when there are so few repetitions

Reply:
You need to scroll upwards to find the variable

Reply:
Yes well what's actually being tested here is the ext_cmdline state, it not being included directly in the call helps to highlight that. Having to scroll up for a grid state we don't really care about doesn't seem too big of a deal to me.

But happy to change this back when I rebase this (and the rest of the changes _are_ wanted).

Reply:
because the _exception_ of using a variable breaks the usual flow of screen tests, and indicates the code is special is some positively meaningful way, like perhaps taking a parametric function argument. It can be worth it when it saves a _lot_ of repeated screens, but should not be used haphazardly just to save 5-6 lines which easily can be eyed to be identical (which is just _one_ of multiple possible reasons a variable could used). 

Code that you can "not care" about should follow the surrounding patterns, not stick out. It will also make tooling to systematically refactor screen expectations (like changing the color scheme in bulk) harder to use as well.